src/core/env.ts

```diff
@@ -87,6 +89,21 @@ function detect(ua: string, env: Env) {
     env.pointerEventsSupported = 'onpointerdown' in window
         && (browser.edge || (browser.ie && +browser.version >= 11));
     env.domSupported = typeof document !== 'undefined';
+
+    const style = document.documentElement.style;
+    env.ieTransform3dSupported = browser.ie && 'transition' in style;
```

Comment:
I'm not sure, but according to the code in https://github.com/apache/echarts/pull/14246/files#diff-323f00b5085e5da04e37591f992fbb0737db751436bec2667c1de6792f82f33dR116

Does `ieTransform3dSupported ` mean only support 2d transform actually? If so, I will suggest using `transformSupported` and `transform3DSupported` to avoid confusing

Reply:
Right. We may need to improve the naming and detection logic.